src/components/navigation.tsx

```diff
@@ -45,7 +45,7 @@ export default function Navigation() {
   const baseUrl = findBaseUrl(pathName);
 
   return (
-    <nav className="flex items-center justify-end space-x-3 border-b-2 border-secondary px-2">
+    <nav className="flex items-center justify-end space-x-3 border-b-2 border-secondary px-2 h-12 w-full">
```

Comment:
if you find yourself adding w-full to all the children, it might be a hint you need to address it on a layout level